package-lock.json

```diff
@@ -18745,10 +18711,12 @@
       }
     },
     "packages/browserslist-config": {
+      "name": "@decidim/browserslist-config",
```

Comment:
I've used the command `npm uninstall social-share-button ` and I see that these lines were added. Did I do something wrong, maybe? 

```
$ npm --version
7.21.1

```

Reply:
This happens because the current `package-lock.json` was probably generated with different NPM version. This started to appear at some point. Don't worry too much about it.

But there is another issue right now which is that the NPM packages cannot be resolved. Can you just delete the `package-lock.json` and then run `npm i` after that. Once that's done, copy the generated `package-lock.json` to the design app.

This is how I've generally done it.

NPM package dependency resolution is not great, which is why this happens. It is just confused which version of the package should be marked as a dependency if multiple packages depend on the same package.

And if this happens again, repeat the same process.

Reply:
Actually I think there might not have even been an issue with this, the CI issue will be fixed after we merge #9041.

Sorry for the confusion!